plugins/workspace-backpack/src/backpack_helpers.js

```diff
@@ -172,10 +191,10 @@ export function registerAllContextMenus(workspace) {
   registerEmptyBackpack(workspace);
```

Comment:
Out of scope for this PR, but we might want to think about how someone would want to only add a subset of these context menu options.

Reply:
Agreed. I have a TODO comment referencing this. I think it is something that someone would want to be able to customize.